### DISCUSSION TITLE
Spacy Architecture

### DISCUSSION THREAD

**Discussion by raffa-dev** [explosion/spaCy#8569] [Language Support]
Hi everyone, 

thanks for the great application and the support. I have a short question which I a was not able to fully resolve.
I have worked with trained models, namely the en_core_lg and ger_news_lg. 

My question is which model Spacy uses for these trained files. I was under the impression that glove is used here but after some research this doesn't seem to be the case. And I was also wondering whether the architectures are identical for English and e.g. German or other languages or if there are adaptations depending on the language which is trained.

Thanks a lot for your help :)

**Comment by polm** [comment]
The pretrained pipelines contain different components, which each have their own architectures. The details are in the docs, like for [the dependency parser](https://spacy.io/api/dependencyparser). They are the same between different languages. 

GloVe vectors are used in the medium and large pretrained pipelines, but they're only one part, and spaCy has no mechanism for training them.
